### ISSUE TITLE
Incorrect title 

### ISSUE THREAD

**Issue by ATXLtheAxolotl** [wrohan/OpenRecSucks#1]
I believe you made a mistake when naming this project I think you meant to put `wrohanSucks` but it’s okay we all make mistakes.

**Comment by GalaxisBeast** [comment]
true. also you can’t just be making repositories to hate on something you can’t even do better yourself
